addons/l10n_it_edi/tests/test_edi_import.py

```diff
@@ -73,28 +74,39 @@ def test_receive_signed_vendor_bill(self):
     def test_receive_same_vendor_bill_twice(self):
```

Comment:
This test generates a `WARNING`[^1], forbidding this PR to be merged.
```
2023-09-19 11:40:06,738 14 INFO 51127691-master-l10n_account odoo.addons.l10n_it_edi.tests.test_edi_import: Starting TestItEdiImport.test_receive_same_vendor_bill_twice ... 
2023-09-19 11:40:06,941 14 WARNING 51127691-master-l10n_account odoo.addons.l10n_it_edi.models.account_move: E-invoice already exists: IT01234567890_FPR02.xml 
```

[^1]: https://runbot.odoo.com/runbot/batch/1171930/build/51127691